src/components/FilterSearch.tsx

```diff
@@ -78,6 +78,8 @@ export interface FilterSearchProps {
   searchOnSelect?: boolean,
   /** A function which is called when a filter is selected. */
   onSelect?: (params: OnSelectParams) => void,
+  /** A function which is called when the input's value changes. This does not replace executeFilterSearch, but gets called just before it */
+  onDropdownInputChange?: (value: string) => void,
```

Comment:
it might be confusing to users that `onSelect` replaces the select behavior but `onDropdownInputChange` is added onto the existing input change behavior. what do you think about changing `onDropdownInputChange` to replace the behavior like `onSelect` does? that would make it more consistent, and gave even more flexibility to users in case they don't want to trigger the `executeFilterSearch` at all